public/js/findMovies.js

```diff
@@ -1,9 +1,11 @@
-const movieName = document.querySelector('#movie-name').value.trim();
-const movieYear = document.querySelector("#movie-year").value.trim();
-const movieGenre = document.querySelector("#movie-genre").value.trim();
+const name = document.querySelector('#movie-name').value.trim();
+const year = document.querySelector("#movie-year").value.trim();
+const genre = document.querySelector("#movie-genre").value.trim();
```

Comment:
I think `genre` can be removed here since we're just returning a link now? We probably won't be supporting search by genre anymore.